features/e2e_tests/e2e_card_payments_tests.feature

```diff
@@ -42,11 +44,20 @@ Feature: E2E Card Payments
     And User will see that "EXPIRATION_DATE" field is highlighted
     And User will see "Invalid field" message under field: "EXPIRATION_DATE"
 
-  @e2e_config_requesttypes_invalid_order
-  Scenario: Unsuccessful payment with config's requestTypes param having values in invalid order
-    Given JS library is configured with REQUEST_TYPES_CONFIG_INVALID_ORDER and BASE_JWT
+  @reactJS
+  Scenario: Successful payment with frictionless card
+    Given JS library is configured with BASIC_CONFIG and BASE_JWT
     And User opens example page
-    When User fills payment form with defined card MASTERCARD_CARD
+    When User fills payment form with defined card VISA_FRICTIONLESS
     And User clicks Pay button
-    Then User will see payment status information: "Invalid field"
-    And User will see that notification frame has "red" color
\ No newline at end of file
+    Then User will see payment status information: "Payment has been successfully processed"
+    And User will see that notification frame has "green" color
+
+  @reactJS
+  Scenario: Successful payment with non-frictionless card
+    Given JS library is configured with BASIC_CONFIG and BASE_JWT
+    And User opens example page
+    When User fills payment form with defined card VISA_NON_FRICTIONLESS
```

Comment:
If you are using VISA_NON_FRICTIONLESS you should also add step for confirm cardinal authentication popup